compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -91,6 +91,11 @@ static llvm::cl::opt<bool> clEnableVectorContractCustomKernels(
                    "LLVMCPUMmt4dVectorLowering pass."),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> clTileDispatchUsingForall(
```

Comment:
This flag seems redundant as the only use of it is to `OR` it with another flag?

Reply:
If someone wants to enable dispatchUsingForAll from command line irrespective of the pipeline -- they can go with this option. This is enabled for default in mmt4d and conv2d pipeline.